sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -112,3 +112,38 @@
     'WriteToJson': 'beam:schematransform:org.apache.beam:json_write:v1'
   config:
     gradle_target: 'sdks:java:extensions:schemaio-expansion-service:shadowJar'
+
+- type: renaming
+  transforms:
+    'ReadFromJdbc': 'ReadFromJdbc'
+    'WriteToJdbc': 'WriteToJdbc'
+  config:
+    mappings:
+      'ReadFromJdbc':
+        driver_class_name: 'driverClassName'
+        jdbc_url: 'jdbcUrl'
+        username: 'username'
+        password: 'password'
```

Comment:
We should seriously think about if there's a better way to store these than in plain text...

Reply:
That's how the transform works today, but I can add a FR for KMS/Secret Manager support. Unless you think it's not worth offering without that support?

Reply:
+1 to a FR at least.